build.gradle

```diff
@@ -17,7 +17,7 @@ import java.text.SimpleDateFormat
 
 buildscript {
     ext {
-        springBootVersion = '3.2.2'
+        springBootVersion = '3.2.4'
```

Comment:
Does this have any effect? With an `ext` I would expect it to be used someplace else in the gradle config but I don't see where that is.

Reply:
Good shout, I don't think it's needed (and it has tripped me up before, where I updated it thinking I was upgrading Spring Boot!)